tests/Behat/UserGraphQLContext/UserGraphQLContext.php

```
@@ -199,65 +213,49 @@
 
-        foreach ($this->responseContent as $fieldName) {
-            Assert::assertArrayHasKey($fieldName, $userData);
-            if (property_exists($this->graphQLInput, $fieldName)) {
-                Assert::assertEquals(
-                    $this->graphQLInput->$fieldName,
-                    $userData[$fieldName]
-                );
-            }
+        if ($decoded === null) {
+            echo 'GraphQL Response content: ' . $content . "\n";
+            throw new \Exception('Invalid JSON response from GraphQL endpoint');
```

PHPMD (2.15.0)

Missing class import via use statement (line '231', column '23'). (Clean Code Rules)

(MissingImport)